docs/source/get-started.md

```diff
@@ -0,0 +1,154 @@
+
+When you change a query and recompile your project - Apollo code generation will rebuild your data model for you. Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection. Future versions of Apollo-Android will also work with AutoValue and other value object generators.
+
+<h2 id="installation">Installation</h2>
+
+The latest Gradle plugin version is [ ![Download](https://api.bintray.com/packages/apollographql/android/apollo-gradle-plugin/images/download.svg) ](https://bintray.com/apollographql/android/apollo-gradle-plugin/_latestVersion)
+
+To use this plugin, add the dependency to your project's `build.gradle` file:
+
+```groovy
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath 'com.apollographql.apollo:apollo-gradle-plugin:x.y.z'
+  }
+}
+
+dependencies {
+  compile 'com.apollographql.apollo:apollo-runtime:x.y.z'
+}
+```
+
+The plugin can then be applied as follows within your app module's `build.gradle` file:
+
+```
+apply plugin: 'com.apollographql.android'
+```
+
+> Note: The Android plugin must be applied before the Apollo plugin
+
+<h2 id="download-schema">Downloading a Schema</h2>
+
+Apollo Android requires a GraphQL schema file as input to the code generation process. A schema file is a JSON file that contains the results of an introspection query. Conventionally this file is called `schema.json`, and you store it next to your `.graphql` files.
+
+You can use `apollo-codegen` to download a GraphQL schema. This works by sending an introspection query to your server:
+
+```sh
+apollo-codegen download-schema http://localhost:8080/graphql --output schema.json
+```
+
+If needed, you can use the `header` option to add additional HTTP headers to the request. For example, to include an authentication token, use `--header "Authorization: Bearer <token>"`:
+
+```sh
+apollo-codegen download-schema https://api.github.com/graphql --output schema.json --header "Authorization: Bearer <token>"
+```
+
+> Note: In the sample project you can find the schema here => `apollo-sample/src/main/graphql/com/apollographql/apollo/sample/schema.json`.
+
+<h2 id="creating-client">Creating a Client</h2>
+
+With the installation complete and schema downloaded, let's create your Apollo Client. In most cases, you’ll want to create a single shared instance of `ApolloClient` and point it at your GraphQL server. `ApolloClient` uses `OkHttp` under the hood for handling network requests. So you will need to create an instance of the `OkHttpClient` and pass it to the `ApolloClient` builder.
+
+```java
+private static final String BASE_URL = "<your_server>/graphql";
+
+OkHttpClient okHttpClient = new OkHttpClient.Builder().build();
```

Comment:
It would be nice to include a blurb about adding an interceptor for header values to your okhttp instance

Reply:
Added blurb and link to the `OkHttpClient` wiki.